lib/pengine/bundle.c

```diff
@@ -1494,6 +1494,50 @@ bundle_print_xml(pe_resource_t *rsc, const char *pre_text, long options,
     free(child_text);
 }
 
+int
+pe__bundle_xml(pcmk__output_t *out, va_list args)
+{
+    long options = va_arg(args, int);
+    resource_t *rsc = va_arg(args, resource_t *);
+    pe__bundle_variant_data_t *bundle_data = NULL;
+    int rc = 0;
+
+    CRM_ASSERT(rsc != NULL);
+
+    get_bundle_variant_data(bundle_data, rsc);
+
+    rc = pe__name_and_nvpairs_xml(out, true, "bundle", 6
+                 , "id", rsc->id
+                 , "type", container_agent_str(bundle_data->agent_type)
+                 , "image", bundle_data->image
+                 , "unique", BOOL2STR(is_set(rsc->flags, pe_rsc_unique))
+                 , "managed", BOOL2STR(is_set(rsc->flags, pe_rsc_managed))
+                 , "failed", BOOL2STR(is_set(rsc->flags, pe_rsc_failed)));
+    CRM_ASSERT(rc == 0);
+    CRM_ASSERT(bundle_data != NULL);
```

Comment:
This last assert isn't needed b/c get_bundle_variant_data() does it already

Reply:
Removed.